config/elasticache/config.go

```diff
@@ -33,6 +33,9 @@ func Configure(p *config.Provider) { //nolint:gocyclo
 			if a, ok := attr["cluster_address"].(string); ok {
 				conn["cluster_address"] = []byte(a)
 			}
+			if a, ok := attr["port"].(string); ok {
```

Comment:
I don't see the port attribute directly in the terraform documentation of this resource, did you test it manually? I think we need to test `examples/elasticache/v1beta1/cluster.yaml`.

Reply:
I have not tested manually, it's not in the Terraform docs but it is in the schema. And you are right, I ran the wrong tests, running correct ones now.